tesseract_core/runtime/core.py

```diff
@@ -145,9 +178,11 @@ def inner(otherfunc: Callable):
     )
 
     @assemble_docstring(api_module.apply)
-    def apply(payload: ApplyInputSchema) -> ApplyOutputSchema:
+    def apply(payload: ApplyInputSchema, job_id: str = "") -> ApplyOutputSchema:
```

Comment:
If I use `str | None`, then `tesseract-runtime check` fails in the python-3.9 test Tesseract. Open to suggestions for improving this!